test/integ/test_put_get_with_gcp_account.py

```diff
@@ -40,6 +42,12 @@ def test_put_get_with_gcp(tmpdir, conn_cnx, db_parameters):
 
     with conn_cnx() as cnx:
         with cnx.cursor() as csr:
+            try:
+                csr.execute(f'ALTER SESSION SET GCS_USE_DOWNSCOPED_CREDENTIAL = {enable_gcs_downscoped}')
+            except ProgrammingError:
+                # skip test if parameter is not available (server not updated)
+                if enable_gcs_downscoped:
+                    return
```

Comment:
Don't swallow this error, please. Remove this and mark the test as `xfail`.

Reply:
Updated